code/game/objects/items/defib.dm

```diff
@@ -276,6 +276,25 @@
 	cooldowncheck()
 	return COMPONENT_DEFIB_STOP
 
+/obj/item/defibrillator/rusted
+	name = "rusted defibrillator"
+	desc = "A old and rusted medical defib, maybe it still works..?"
+	icon_state = "defibrusted"
+	inhand_icon_state = "defibrusted"
+	worn_icon_state = "defibunit-rusted"
+	lefthand_file = 'icons/mob/inhands/equipment/medical_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/equipment/medical_righthand.dmi'
+	powered_state = "defibrusted-powered"
+	safety = 0
```

Comment:
```suggestion
	safety = FALSE
```

This is what the parent defibrillator has:
```dm
	/// If you can zap people with the defibs on harm mode
	var/safety = TRUE
```

Use the Boolean macro for Boolean values

Reply:
thanks, i knew i should of switched it to FALSE but i was focusing on other things at the time and forgot, sorry!